.github/workflows/test.yml

```
@@ -7,50 +7,51 @@ on:
-      - uses: golangci/golangci-lint-action@v9
+      - uses: golangci/golangci-lint-action@63c23d5020da8f97005b6de340504130bdbe42c7 # v9
         with:
           args: -v
           skip-cache: true
+
   build:
     needs: lint
     strategy:
         matrix:
-            os: [ ubuntu-latest ]
             goos: [ linux ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)